[debug] [2020-04-21T15:43:57.830Z] ----------------------------------------------------------------------
[debug] [2020-04-21T15:43:57.832Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\franc\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js functions:config:get
[debug] [2020-04-21T15:43:57.832Z] CLI Version:   8.1.1
[debug] [2020-04-21T15:43:57.832Z] Platform:      win32
[debug] [2020-04-21T15:43:57.832Z] Node Version:  v12.14.0
[debug] [2020-04-21T15:43:57.833Z] Time:          Tue Apr 21 2020 12:43:57 GMT-0300 (Argentina Standard Time)
[debug] [2020-04-21T15:43:57.833Z] ----------------------------------------------------------------------
[debug] [2020-04-21T15:43:57.833Z] 
[debug] [2020-04-21T15:43:57.840Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-21T15:43:57.858Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (C:\Users\franc\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:160:19)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async GoogleAuth.getClient (C:\Users\franc\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:502:17)
    at async GoogleAuth.getAccessToken (C:\Users\franc\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:524:24)
[error] 
[error] Error: An unexpected error has occurred.
